apps/drupal/src/ParticleTools/ParticleTools.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Drupal\particle\ParticleTools;
+
+/**
+ * Provides utility functions for all preprocessors.
+ *
+ * This approach is influenced by Adventist's theme tools system.
+ * For advanced examples including implementing custom services,
+ *
+ * @see https://bitbucket.org/phase2tech/adventist/src/develop/src/themes/ahs_theme/src/ThemeTools.php
```

Comment:
Don't put a link to a private bitbucket in the public particle project.

Reply:
Would rather see the approach here documented somewhere in the Particle docs and linking to that.  Also a basic summary of what this ParticleTools is for.

Reply:
Oh, and I see there is a README file below, so just link to that and put docs about "advanced examples" there.